### ISSUE TITLE
Global configuration option for Athena workgroup

### ISSUE THREAD

**Issue by vuchetichbalint** [awslabs/aws-data-wrangler#437]
I'm always use `wr.athena.read_sql_table()` with a specific workgroup, while other teams may use with a different one (for the very same query). Right now there is no option to set a global default for each user.

Workaround:
* Everyone set in their own `.bashrc`:
  * `export AWS_ATHENA_DEFAULT_WORKGROUP=specific_athena_workgroup`
* For every query set workgroup _explicitly_:
  * `wr.athena.read_sql_table(..., workgroup=os.environ['AWS_ATHENA_DEFAULT_WORKGROUP'])` 

But that is far from ideal, it would be much better if it could be managed by `_config.py`, and later use inplicitly.

**Comment by igorborgest** [comment]
Thank you @vuchetichbalint 

I've just added the workgroup to our global configs, so you can use it programmatically:

```py3
wr.config.workgroup = "my_workgroup"
df = wr.athena.read_sql_query(sql="SELECT ...", database=glue_database)
```

OR you can define it through the `WR_WORKGROUP` environment variable:

```sh
export WR_WORKGROUP=my_workgroup
```

```py3
df = wr.athena.read_sql_query(sql="SELECT ...", database=glue_database)
```

Do you mind to install it directly from our repository and check if it is everything fine for your use case before the official release?
The idea is to publish it in the version `1.10.0` this weekend.

```sh
pip install git+https://github.com/awslabs/aws-data-wrangler.git --no-use-pep517
```

**Comment by igorborgest** [comment]
Released on [1.10.0](https://github.com/awslabs/aws-data-wrangler/releases/tag/1.10.0)
